components/datetime/src/provider/calendar/symbols.rs

```diff
@@ -118,14 +123,7 @@ macro_rules! symbols {
 symbols!(
     months,
     pub struct SymbolsV1<'data>(
-        #[cfg_attr(
-            feature = "serde",
-            serde(
-                borrow,
-                deserialize_with = "icu_provider::serde::borrow_de_utils::array_of_cow"
-            )
-        )]
-        pub [Cow<'data, str>; 12],
+        #[cfg_attr(feature = "serde", serde(borrow))] pub ZeroMap<'data, MonthCode, str>,
```

Comment:
Thought: I hadn't considered the data size increase of this. Should we consider mitigating it by making `SymbolsV1` be an enum between the 12-month cow, like we had before, and the zeromap? Slight increase in code size but likely a more significant win in data size.

Reply:
hmmm, we could, yes

Reply:
Just realized that we've been omitting the 13th month for 13 month cals all along, oops